tests/serverpod_new_auth_test/serverpod_new_auth_test_server/lib/src/web/routes/root.dart

```diff
@@ -1,11 +1,12 @@
 import 'dart:io';
 
 import 'package:serverpod/serverpod.dart';
-import 'package:serverpod_new_auth_test_server/src/web/widgets/built_with_serverpod_page.dart';
+import 'package:serverpod_new_auth_test_server/src/web/components/built_with_serverpod_page.dart';
 
-class RouteRoot extends WidgetRoute {
+class RouteRoot extends ComponentRoute {
```

Comment:
```suggestion
class RootComponentRoute extends ComponentRoute {
```

Or `RootRoute`? Because usually the "base class" is at the end.

Reply:
(I think there's also another one just like this I just read over…)

Reply:
I think `RootRoute` still makes sense here, no?